CI_DailyBuildUpdates.py

```diff
@@ -13,15 +13,19 @@ def main():
     parser = argparse.ArgumentParser(description='Get the daily buid updates')
     parser.add_argument('--info_type', default='brief', choices=['brief','detailed'], help='specify the job info type (brief or detailed)')
     parser.add_argument('--zone', help='specify the lease/zone', type= lambda arg:arg.split(','))
-    parser.add_argument('--ci_arch', default='p', choices=['p','z'], help='Specify the CI architecture type (p or z), default is p')
+    parser.add_argument('--ci_arch', default='p', choices=['p','z','a'], help='Specify the CI architecture type (p or z or a), default is p')
```

Comment:
Should we mention it as an arch ? Since those are just different jobs of Power architecture